lib/Conversion/TorchToTcp/Elementwise.cpp

```diff
@@ -74,6 +74,45 @@ class ConvertAtenAddSubOp : public OpConversionPattern<AtenOpT> {
   }
 };
 
+template <typename AtenOpT, typename TcpOpT>
+class ConvertAtenMinMaxOp : public OpConversionPattern<AtenOpT> {
```

Comment:
In addition to creating this class, we also need to add it to the patterns in the main function below.

See the last line in the file, which does the following for div op.
```
patterns.add<ConvertAtenDivFOp>(typeConverter, context);
```